src/runtime_stack_check.js

```diff
@@ -29,7 +29,7 @@ if (!ENVIRONMENT_IS_PTHREAD) {
   HEAP32[0] = 0x63736d65; /* 'emsc' */
 #if USE_PTHREADS
 } else {
-  if (HEAP32[0] !== 0x63736d65) throw 'Runtime error: The application has corrupted its heap memory area (address zero)!';
```

Comment:
Is this unrelated?

Reply:
Yeah, not directly related but an annoying thing I fixed while I noticed it here.